### PR TITLE
Adding build script for  generic-admission-server

### PR DESCRIPTION
**Pull request by asellappen**
[root@d57a132ee1f0 ~]# sh generic-admission-server_ubi_8.3.sh
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 3:05:56 ago on Fri Oct  1 06:48:38 2021.
Package git-2.27.0-1.el8.ppc64le is already installed.
Package make-1:4.2.1-10.el8.ppc64le is already installed.
Package golang-1.15.14-2.module+el8.4.0+12542+e3fec473.ppc64le is already installed.
Dependencies resolved.
Nothing to do.
Complete!
Cloning into 'generic-admission-server'...
remote: Enumerating objects: 32345, done.
remote: Counting objects: 100% (3015/3015), done.
remote: Compressing objects: 100% (2204/2204), done.
remote: Total 32345 (delta 670), reused 2922 (delta 606), pack-reused 29330
Receiving objects: 100% (32345/32345), 37.41 MiB | 19.50 MiB/s, done.
Resolving deltas: 100% (14621/14621), done.
Already on 'master'
Your branch is up to date with 'origin/master'.
 found to checkout
GO111MODULE=on GOPROXY=https://proxy.golang.org go build -o _output/bin/generic-admission-server github.com/openshift/generic-admission-server/pkg/cmd
=== RUN   TestV1Beta1Webhook
W1001 06:59:45.858070   10113 authorization.go:47] Authorization is disabled
W1001 06:59:45.858180   10113 authentication.go:40] Authentication is disabled
=== RUN   TestV1Beta1Webhook/test_validator
=== RUN   TestV1Beta1Webhook/test_mutator
--- PASS: TestV1Beta1Webhook (0.01s)
    --- PASS: TestV1Beta1Webhook/test_validator (0.00s)
    --- PASS: TestV1Beta1Webhook/test_mutator (0.00s)
=== RUN   TestV1Webhook
W1001 06:59:45.864466   10113 authorization.go:47] Authorization is disabled
W1001 06:59:45.864522   10113 authentication.go:40] Authentication is disabled
=== RUN   TestV1Webhook/test_validator
body is {"kind":"AdmissionReview","apiVersion":"admission.k8s.io/v1","request":{"uid":"","kind":{"group":"","version":"","kind":"TestKind"},"resource":{"group":"","version":"","resource":""},"operation":"","userInfo":{},"object":null,"oldObject":null,"options":null},"response":{"uid":"","allowed":true}}

=== RUN   TestV1Webhook/test_mutator
body is {"kind":"AdmissionReview","apiVersion":"admission.k8s.io/v1","request":{"uid":"","kind":{"group":"","version":"","kind":"TestKind"},"resource":{"group":"","version":"","resource":""},"operation":"","userInfo":{},"object":null,"oldObject":null,"options":null},"response":{"uid":"","allowed":true,"patch":"e30="}}

--- PASS: TestV1Webhook (0.00s)
    --- PASS: TestV1Webhook/test_validator (0.00s)
    --- PASS: TestV1Webhook/test_mutator (0.00s)
PASS
ok      github.com/openshift/generic-admission-server/pkg/apiserver     (cached)
?       github.com/openshift/generic-admission-server/pkg/cmd   [no test files]
?       github.com/openshift/generic-admission-server/pkg/cmd/server    [no test files]
?       github.com/openshift/generic-admission-server/pkg/registry/admissionreview      [no test files]
 Test Success
[root@d57a132ee1f0 ~]#

### CODE REVIEW

**Comment by seth-priya** [comment]
@gururajrkatti please review

File: g/generic-admission-server/generic-admission-server_ubi_8.3.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : generic-admission-server
+# Version               : master
+# Note                  : glide install required for v1.14.0 & previous versions
```
Comment (gururajrkatti) [inline g/generic-admission-server/generic-admission-server_ubi_8.3.sh:5]: Hi @seth-priya Is this kind of comment ok in the header ?

File: g/generic-admission-server/generic-admission-server_ubi_8.3.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+cd  generic-admission-server
+git checkout $VERSION
+ret=$?
+if [ $ret -eq 0 ] ; then
+ echo "$Version found to checkout "
+else
+ echo "$Version not found "
+ exit
+fi
+#verify build only for latest versions
```
Comment (gururajrkatti) [inline g/generic-admission-server/generic-admission-server_ubi_8.3.sh:52]: Does it need any checks for latest version here ?

File: g/generic-admission-server/generic-admission-server_ubi_8.3.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+make build
+ret=$?
+if [ $ret -ne 0 ] ; then
+	echo " build failed "
+else
+    go test -v ./...
+	ret=$?
+    if [ $ret -ne 0 ] ; then
+	  echo " Test failed "
+	else
```
Comment (gururajrkatti) [inline g/generic-admission-server/generic-admission-server_ubi_8.3.sh:62]: Indentation issue

Comment (asellappen) [reply g/generic-admission-server/generic-admission-server_ubi_8.3.sh]: taken care now tks.

**Pull request merged**